### PR TITLE
Added Dockerfile

### PR DESCRIPTION
**Pull request by andreikaigorodov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,4 +20,16 @@ For more information, [visit the Scalatron web site here on Github](http://scala
 
 ## License
 
-Scalatron is licensed under the Creative Commons Attribution 3.0 Unported License. The documentation, tutorial and source code are intended as a community resource and you can basically use, copy and improve them however you want. Included works are subject to their respective licenses. 
+Scalatron is licensed under thie Creative Commons Attribution 3.0 Unported License. The documentation, tutorial and source code are intended as a community resource and you can basically use, copy and improve them however you want. Included works are subject to their respective licenses.
```
Comment (jmhofer) [inline README.md]: you accidentally added an "i" in there ("thie"). 😛

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+FROM registry.opensource.zalan.do/stups/openjdk:8-53
```
Comment (jcranky) [inline Dockerfile:1]: Is this accessible for non-zalando people?

**Comment by jmhofer** [comment]
maybe use sbt-docker or sbt-native-packager for this?

**Comment by jcranky** [comment]
@jmhofer good point

File: Dockerfile (hunk 2 above)
Comment (andreikaigorodov) [reply Dockerfile:1]: yes, it is accessible. Many zalando open source projects use this registry.

File: README.md (hunk 1 above)
Comment (andreikaigorodov) [reply README.md]: indeed, fixed.

**Review by jmhofer — APPROVED** [review]
ok, let's do the sbt-docker thing separately

**Pull request merged**